post_installation_configuration/configuring-private-cluster.adoc

```diff
@@ -10,6 +10,10 @@ After you install an {product-title} version {product-version} cluster, you can
 
 include::modules/private-clusters-about.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+For more information about making external load balancers internal, see xref:../networking/nw-ingress-controller-endpoint-publishing-strategies.adoc#nw-ingresscontroller-change-internal[Configuring the Ingress Controller endpoint publishing scope to Internal].
```

Comment:
@lihongan thanks for your feedback. I think the topic you linked in your Jira comment can be easily linked here for quick customer reference. However, it is harder to create this link for the private-clusters-default.adoc module because it is included in many assemblies, so I would need to add this link in all the Cloud install assemblies. I'm not sure I want to make that deep of a cut into the docs 😬 Unfortunately, we are only able to create links in assemblies otherwise it makes the most sense to add the link to a module